Dockerfile

```diff
@@ -71,12 +71,13 @@ RUN cd /rapidpro && bower install --allow-root
 
 # Install `psql` command (needed for `manage.py dbshell` in stack/init_db.sql)
 # Install `libmagic` (needed since rapidpro v3.0.64)
-RUN apk add --no-cache postgresql-client libmagic
+# Install `supervisor` to run a celery queue in same container for uploads
+RUN apk add --no-cache postgresql-client libmagic supervisor
```

Comment:
If you want supervisor, you want to `pip install supervisor` rather as the base image uses a version of Python separate from the system Python which is not installed. When you `apk add supervisor`, you install _another_ Python as a dependency.